documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -37,15 +37,15 @@ Set the `disable_commandline_tags` command to `1` to stop `start.elf` from filli
 
 === `kernel`
 
-`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Pi 1, Pi Zero, and Compute Module is `kernel.img`, and on the Pi 2, Pi 3, and Compute Module 3 it is `kernel7.img`. On the Pi4, it is `kernel7l.img`.
+`kernel` is the alternative filename on the boot partition to use when loading the kernel. The default value on the Raspberry Pi 1, Zero and Zero W, and Raspberry Pi Compute Module 1 is `kernel.img`, and on the Raspberry Pi 2, 3, 3+ and Zero 2 W, and Raspberry Pi Compute Modules 3 and 3+ it is `kernel7.img`. On the Raspberry Pi 4 and 400, and Raspberry Pi Compute Module 4 it is `kernel7l.img`.
```

Comment:
In hindsight, this sentence is now so long that it probably deserves to be broken up a bit? How about:
"The default value on the Raspberry Pi 1, Zero and Zero W, and Raspberry Pi Compute Module 1 is `kernel.img`. The default value on the Raspberry Pi 2, 3, 3+ and Zero 2 W, and Raspberry Pi Compute Modules 3 and 3+ is `kernel7.img`. The default value on the Raspberry Pi 4 and 400, and Raspberry Pi Compute Module 4 is `kernel7l.img`."
?

Reply:
@lurch 
I split it into small pieces. However, overall, the sentence became a little longer.

Reply:
A little longer, but also a little clearer, and that's the most important thing :slightly_smiling_face:

Reply:
@lurch 
I think I have learned  a lot.  Thanks